opencensus/exporters/trace/ocagent/ocagent_exporter.h

```diff
@@ -0,0 +1,58 @@
+// Copyright 2019, OpenCensus Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef OPENCENSUS_EXPORTERS_TRACE_OCAGENT_OCAGENT_EXPORTER_H_
+#define OPENCENSUS_EXPORTERS_TRACE_OCAGENT_OCAGENT_EXPORTER_H_
+
+#include <string>
+
+#include "absl/base/macros.h"
+#include "absl/strings/string_view.h"
+#include "absl/time/time.h"
+
+#include "opencensus/proto/agent/trace/v1/trace_service.grpc.pb.h"
+#include "opencensus/proto/agent/trace/v1/trace_service.pb.h"
+
+namespace opencensus {
+namespace exporters {
+namespace trace {
+
+struct OcAgentOptions {
+  // The OcAgent address to use.
+  std::string address;
+
+  // The RPC deadline to use when exporting to OcAgent.
+  absl::Duration rpc_deadline = absl::Seconds(5);
+
+  // (optional) By default, the exporter connects to Stackdriver using gRPC. If
+  // this stub is non-null, the exporter will use this stub to send gRPC calls
+  // instead. Useful for testing.
+  std::unique_ptr<opencensus::proto::agent::trace::v1::TraceService::Stub>
```

Comment:
Stub -> StubInterface so we can mock it later. I missed this when I was changing the Stackdriver exporters. :(((

https://github.com/census-instrumentation/opencensus-cpp/pull/366